sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>com.azure.spring</groupId>
   <artifactId>azure-spring-boot-bom</artifactId>
-  <version>3.6.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-bom;dependency} -->
+  <version>3.5.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot-bom;dependency} -->
```

Comment:
We already released [azure-spring-boot-bom:3.6.0](https://mvnrepository.com/artifact/com.azure.spring/azure-spring-boot-bom/3.6.0), why this PR recover to 3.5.0?

Reply:
Hi, @moarychan , could you please fix this? Related PR: https://github.com/Azure/azure-sdk-for-java/pull/20968/files